runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -30,16 +55,32 @@ public SequencerView(CorfuRuntime runtime) {
      * @param numTokens The number of tokens to reserve.
      * @return The first token retrieved.
      */
+    @Deprecated
     public TokenResponse nextToken(Set<UUID> streamIDs, int numTokens) {
-        return layoutHelper(e -> CFUtils.getUninterruptibly(e.getPrimarySequencerClient()
-                .nextToken(streamIDs, numTokens)));
+        if (numTokens == 0) {
+            return currentToken(streamIDs.toArray(new UUID[0]));
+        } else if (numTokens == 1) {
+            return nextToken(streamIDs.toArray(new UUID[0]));
+        }
+        throw new UnrecoverableCorfuError("numTokens other than 0,1 no longer supported");
     }
 
 
+    @Deprecated
     public TokenResponse nextToken(Set<UUID> streamIDs, int numTokens,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)